charts/vector/values.yaml

```
@@ -1,75 +1,75 @@
+# "Agent", "Aggregator", and "Stateless-Aggregator".
+
+# Each role is created with the following workloads:
+# Agent = DaemonSet
+# Aggregator = StatefulSet
+# Stateless-Aggregator = Deployment
 role: "Aggregator"
 
-# rollWorkload -- Add a checksum of the generated ConfigMap to workload annotations
+# rollWorkload -- Add a checksum of the generated ConfigMap to workload annotations.
 rollWorkload: true
 
-# commonLabels -- Add additional labels to all created resources
+# commonLabels -- Add additional labels to all created resources.
 commonLabels: {}
 
-## Define the Vector image to use
+# Define the Vector image to use.
 image:
-  # image.repository -- Override default registry + name for Vector
+  # image.repository -- Override default registry and name for Vector's image.
   repository: timberio/vector
-  # image.pullPolicy -- Vector image pullPolicy
+  # image.pullPolicy -- The [pullPolicy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) for
+  # Vector's image.
   pullPolicy: IfNotPresent
-  # image.pullSecrets -- Agent repository pullSecret (ex: specify docker registry credentials)
-  ## Ref: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+  # image.pullSecrets -- The [imagePullSecrets](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod)
+  # to reference for the Vector Pods.
   pullSecrets: []
-  # image.tag -- Vector image tag to use
-  # @default -- Chart's appVersion
+  # image.tag -- The tag to use for Vector's image.
+  # @default -- Derived from the Chart's appVersion.
   tag: ""
-  # image.sha -- Vector image sha to use
+  # image.sha -- The SHA to use for Vector's image.
   sha: ""
 
-# replicas -- Set the number of Pods to create
-## Valid for Aggregator and Stateless-Aggregator
+# replicas -- Specify the number of Pods to create. Valid for the "Aggregator" and "Stateless-Aggregator" roles.
 replicas: 1
 
-# podManagementPolicy -- Specify the podManagementPolicy for the Aggregator role
-## Valid for Aggregator role
+# podManagementPolicy -- Specify the [podManagementPolicy](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies)
+# for the StatefulSet. Valid for the "Aggregator" role.
 podManagementPolicy: OrderedReady
 
-## Create a Secret resource for Vector to use
+# Create a Secret resource for Vector to use.
 secrets:
-  # secrets.generic -- Each Key/Value will be added to the Secret's data key, each value should be raw and NOT base64 encoded
-  ## Any secrets can be provided here, it's commonly used for credentials and other access related values.
-  ## NOTE: Don't commit unencrypted secrets to git!
+  # secrets.generic -- Each Key/Value will be added to the Secret's data key, each value should be raw and NOT base64
+  # encoded. Any secrets can be provided here, and it's commonly used for credentials and other access related values.
```

nit but I'd make this two sentences.

```suggestion
  # encoded. Any secrets can be provided here. It's commonly used for credentials and other access related values.
```

---

Changed in c725ba6